o2.sh

```diff
@@ -76,6 +77,7 @@ cmake $SOURCEDIR -DCMAKE_INSTALL_PREFIX=$INSTALLROOT
       ${GSL_ROOT:+-DGSL_DIR=$GSL_ROOT}                            \
       ${PYTHIA_ROOT:+-DPYTHIA8_INCLUDE_DIR=$PYTHIA_ROOT/include}  \
       ${CMAKE_BUILD_TYPE:+-DCMAKE_BUILD_TYPE=$CMAKE_BUILD_TYPE}   \
+      -DMS_GSL_INCLUDE_DIR=$MS_GSL_ROOT/include                   \
```

Comment:
Is O2 already ready to accept this?

Reply:
No test will be broken. This is just a definition of some variable ... which if not used will do nothing.

Reply:
ok. That's what I meant with my first question.

Reply:
I have the version that used this ready. Just needs some cleanup.

Reply:
For the record: Here is the pull request that used it on the O2 side: https://github.com/AliceO2Group/AliceO2/pull/553